src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderDropdown.scss

```diff
@@ -51,9 +51,19 @@
   background-color: rgba(241, 241, 243, 0.1);
 }
 
-.global-header--align-center {
+.global-header--main-dropdown-item {
   display: flex;
   align-items: center;
+  width: 100%;
+}
+
+.global-header--main-dropdown-item-link {
```

Comment:
The `!important`s are needed to force an override of the `<Link>` style that is otherwise present.

Reply:
minor, but how's about alphabetizing this selector list?

Reply:
you got it - I'll reorder.